flight_manager.py

```diff
@@ -0,0 +1,111 @@
+"""Controller file to starting flight process"""
+import argparse
+import logging
+
+from multiprocessing import Process, Queue
+from multiprocessing.managers import BaseManager
+
+from logger import init_logger, worker_configurer
+from communication import Communication
+from flight.flight import flight
+from flight.state_settings import StateSettings
+
+
+class FlightManager:
+    def __init__(self, state_settings: StateSettings) -> None:
+        """
+        Constructor for the flight manager
+
+        Parameters
+        ----------
+            state_settings: StateSettings
+                Settings for flight state machine
+        """
+        self.state_settings: StateSettings = state_settings
+
+    def main(self) -> None:
+        """
+        Central function to run all threads of state machine
+        """
+        parser: argparse.ArgumentParser = argparse.ArgumentParser()
+        parser.add_argument("-s", "--simulation", help="using the simulator", action="store_true")
+        args = parser.parse_args()
+        logging.debug("Simulation flag %s", "enabled" if args.simulation else "disabled")
+        self.run_threads(args.simulation)
+
+    def init_flight(
+        self, flight_args: tuple[Communication, bool, Queue[str], StateSettings]
+    ) -> Process:
+        """
+        Initializes the flight state machine process
+
+        Parameters
+        ----------
+        flight_args: Tuple[Communication, bool, Queue, StateSettings]
+            Arguments necessary for flight logging and state machine settings
+        Returns
+        -------
+        Process
+            Multiprocessing object for flight state machine
+        """
+        return Process(target=flight, name="flight", args=flight_args)
+
+    def run_threads(self, sim: bool) -> None:
+        """
+        Runs the various threads present in the state machine
+
+        Parameters
+        ----------
+            sim: bool
+                Decides if the simulator is active
```

Comment:
Not sure why this has an extra indent
```suggestion
        Parameters
        ----------
        sim: bool
            Decides if the simulator is active
```